### ISSUE TITLE
Fix issue with imports

### ISSUE THREAD

**Issue by jchodera** [choderalab/bhmm#5]
I am debugging an issue with the latest version of the code

```
[LSKI1497:bhmm.choderalab/examples/p5ab-hairpin] choderaj% python generate-figure-twostate.py
Traceback (most recent call last):
  File "generate-figure-twostate.py", line 7, in <module>
    from bhmm import testsystems
  File "/Users/choderaj/anaconda/lib/python2.7/site-packages/bhmm-0.1.0-py2.7-macosx-10.5-x86_64.egg/bhmm/__init__.py", line 14, in <module>
    from mlhmm import MLHMM
  File "/Users/choderaj/anaconda/lib/python2.7/site-packages/bhmm-0.1.0-py2.7-macosx-10.5-x86_64.egg/bhmm/mlhmm.py", line 14, in <module>
    from ml.baum_welch import BaumWelchHMM
  File "/Users/choderaj/anaconda/lib/python2.7/site-packages/bhmm-0.1.0-py2.7-macosx-10.5-x86_64.egg/bhmm/ml/baum_welch.py", line 6, in <module>
    import bhmm.hidden as hidden
  File "/Users/choderaj/anaconda/lib/python2.7/site-packages/bhmm-0.1.0-py2.7-macosx-10.5-x86_64.egg/bhmm/hidden/__init__.py", line 3, in <module>
    from .api import *
  File "/Users/choderaj/anaconda/lib/python2.7/site-packages/bhmm-0.1.0-py2.7-macosx-10.5-x86_64.egg/bhmm/hidden/api.py", line 12, in <module>
    import impl_python as ip
```

**Comment by jchodera** [comment]
Submitted a PR (#6) to fix this.

**Comment by franknoe** [comment]
Anything I can do here? For me this issue does not come up.

**Comment by jchodera** [comment]
This was fixed by #8.
